### ISSUE TITLE
Tamer not working with python 3.11.2 (Windows 10)

### ISSUE THREAD

**Issue by bussa92** [aiplan4eu/unified-planning#331]
**Describe the bug**
Installation of pytamer not possible with python 3.11.2

**To Reproduce**
1. pip install --pre unified-planning
2. pip install --pre unified-planning[tamer]
3. C:\Python\Python311\Lib\site-packages\unified_planning-0.5.0.169.dev1.dist-info was overwritten to older version which doesn´t support tamer and pyperplan

**Comment by bussa92** [comment]
Yes, we were able to use Tamer with a downgraded version of python.

**Comment by alvalentini** [comment]
Last version of Tamer is compiled up to Python 3.11 on windows.
